### PR TITLE
optimize setup

### PR DESCRIPTION
**Pull request by LawyZheng**
1. `orgs.name` created in `secrets.toml` is `Skyvern`, not consistent with  orgs.name in database
2. updating LLM provider will re-init my local dev env. So I depart `setup.sh` into three parts: env, dev dependencies, database. Default is to init all, but we can only init the part we want in some cases by adding the second param like `setup.sh [env|dev|database]` 

For wise, I think the setup needs to be refactored by Python scripts or something else like CLI:
1. bash is not well-performed in Windows
2. bash is not readable enough

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -250,24 +250,52 @@ create_organization() {
     fi
 
     # Update the secrets-open-source.toml file
-    echo -e "[skyvern]\nconfigs = [\n    {\"env\" = \"local\", \"host\" = \"http://0.0.0.0:8000/api/v1\", \"orgs\" = [{name=\"Skyvern\", cred=\"$api_token\"}]}\n]" > .streamlit/secrets.toml
+    echo -e "[skyvern]\nconfigs = [\n    {\"env\" = \"local\", \"host\" = \"http://0.0.0.0:8000/api/v1\", \"orgs\" = [{name=\"Skyvern-Open-Source\", cred=\"$api_token\"}]}\n]" > .streamlit/secrets.toml
```
Comment (suchintan) [inline setup.sh:253]: Can we keep this as `Skyvern`?

Comment (LawyZheng) [reply setup.sh:253]: sure, but I'm confused why we regist `org.name="Skyvern-Open-Source"` but get `orgs.name="Skyvern"` written in the sercretes.toml?

Comment (suchintan) [reply setup.sh:253]: The orgs.name inside streamlit is cosmetic -- it doesn't map to anything!

Comment (LawyZheng) [reply setup.sh:253]: aha, ok. I will revert this commit

**Pull request closed without merging**